### PR TITLE
add seqwish

### PR DESCRIPTION
**Pull request by bgruening**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/seqwish/seqwish.xml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    #end if
+    #if $B:
+        -B $B
+    #end if
+
+    ]]>
+    </command>
+
+    <inputs>
+        <param argument="--paf-alns" name="paf_alns" type="data" format="paf" label="Induce the graph from these PAF formatted alignments" />
```
Comment (nsoranzo) [inline tools/seqwish/seqwish.xml:33]: I don't think we have a `paf` datatype? The [minimap2 wrapper](https://github.com/galaxyproject/tools-iuc/blob/master/tools/minimap2/minimap2.xml#L263) uses `tabular`, but maybe we should add it?

Comment (bgruening) [reply tools/seqwish/seqwish.xml:33]: Mh, yes maybe we should add it. Do you want me to put here "tabular" for the meantime and until the next Galaxy release (with new datatype) is out?

Comment (nsoranzo) [reply tools/seqwish/seqwish.xml:33]: I think PRs adding datatypes to a stable release are acceptable, so we could target 20.01?

Comment (bgruening) [reply tools/seqwish/seqwish.xml:33]: We are still running 19.09. But the earlier we get those datatypes in the better imho.

File: tools/seqwish/seqwish.xml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            help="Limit transitive closure to include no more than N copies of a given input base" />
+        <param argument="-k" type="integer" min="1" optional="true" label="Filter exact matches below this length"
+            help="This can smooth the graph locally and prevent the formation of complex local graph topologies from forming due to differential alignments." />
+        <param argument="-B" type="integer" min="1" optional="true" label="Number of bp to use for transitive closure batch"
+            help="" />
+
+    </inputs>
+
+    <outputs>
+        <data name="out_gfa" format="gfa" />
```
Comment (nsoranzo) [inline tools/seqwish/seqwish.xml:48]: Also `gfa` is not a recognised datatype, I can try to add both.

Comment (nsoranzo) [reply tools/seqwish/seqwish.xml]: https://github.com/galaxyproject/galaxy/pull/9592

**Comment by bgruening** [comment]
Thanks @nsoranzo

**Pull request merged**